app/javascript/components/patient/household/utils/IconMinor.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
```

Comment:
Why is this a separate component? it's only used in one place and is minimal code?

Reply:
This should probably be added to the ApplyToHousehold table as well, so keep this as is.  I refactored this code as a part of my PR so whoever gets in last will have to update that code

Reply:
This now needs to be added to the `HouseholdMemberTable` component